dotCMS/src/main/java/com/dotmarketing/portlets/rules/util/RulesImportExportUtil.java

```diff
@@ -83,69 +68,69 @@ public void export(File file) throws IOException {
 		}
 	}
 
-	/**
-	 * 
-	 * @param file
-	 * @throws IOException
-	 */
-	public void importRules(File file) throws IOException {
+	public void importRules(final File file) throws IOException {
 
-		RulesAPI rapi = APILocator.getRulesAPI();
+		final ObjectMapper mapper = new ObjectMapper();
+		final StringWriter stringWriter = new StringWriter();
+
+		try (BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(file.toURI()))) {
 
-		BufferedReader in = null;
-		try {
-			User user = APILocator.getUserAPI().getSystemUser();
-			FileReader fstream = new FileReader(file);
-			in = new BufferedReader(fstream);
-			ObjectMapper mapper = new ObjectMapper();
 			mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-			StringWriter sw = new StringWriter();
+
 			String str;
-			while ((str = in.readLine()) != null) {
-				sw.append(str);
+			while ((str = bufferedReader.readLine()) != null) {
+				stringWriter.append(str);
 			}
 
-			RulesImportExportObject importer = mapper.readValue(sw.toString(), RulesImportExportObject.class);
+			RulesImportExportObject importer = mapper.readValue
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'importer' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4925354)